### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by akaDuality**
Accidentally done several different tasks in single PR:

- [x] Fix text recognition for elements
- [x] Select container after grouping
- [x] Move more code to DocumentPresenter. It 
- [x] Fix several undos of drawing actions
- [x] Can delete any element by backspace button
- [x] Rework windows managment
- [x] Undo element deletion inside container
- [x] Undo container deletion
- [x] Define interface of DocumentPresenter – it manages `controls` changes

### CODE REVIEW

File: Shared/Sources/Canvas/CanvasPresenter.swift [hunk 1]
```diff
@@ -170,20 +157,34 @@ public class CanvasPresenter: DocumentPresenter {
...
         ui.drawnControls.first { control in
-            control.model?.frame == model.frame
+            control.model === model
         }
     }
 }
+
+// MARK: - Undo
+extension CanvasPresenter {
+    func registeUndo(for action: Undoable) {
```
Comment (drucelweisse) [inline Shared/Sources/Canvas/CanvasPresenter.swift:180]: just typo

File: VoiceOver Designer/Features/Tests/NavigatorTests/EmptyTests.swift [hunk 2]
```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mikhail Rubanov on 30.12.2022.
+//
+
+import Foundation
```
Comment (drucelweisse) [inline VoiceOver Designer/Features/Tests/NavigatorTests/EmptyTests.swift:8]: It seems to be an empty file or it doesn't have tests yet?

**Review by drucelweisse — CHANGES_REQUESTED** [review]
Loved how DocumentPresenter become shared entry point to manage controls. And an undoable protocol

File: VoiceOver Designer/Features/Tests/NavigatorTests/EmptyTests.swift (hunk 2 above)
Comment (akaDuality) [reply VoiceOver Designer/Features/Tests/NavigatorTests/EmptyTests.swift:8]: I moved all files out of this target, but a target can't exists without files inside. I had two ways: remove target at all or add missing file. I decide to keep target exists to simplify feature tests

**Review by drucelweisse — APPROVED** [review]

**Pull request merged**